### PR TITLE
Update pylab example to OO format

### PR DESCRIPTION
**Pull request by dmgt**
## PR Summary

Updating this example to use object oriented format as opposed to previous pyplot API.

## PR Checklist

- [x] Code is PEP 8 compliant 
- [x] Documentation is sphinx and numpydoc compliant

### CODE REVIEW

**Comment by choldgraf** [comment]
@NelleV could you give a quick review?

File: examples/pylab_examples/log_test.py [hunk 1]
```diff
@@ -10,10 +10,13 @@
...
 dt = 0.01
 t = np.arange(dt, 20.0, dt)
 
-plt.semilogx(t, np.exp(-t/5.0))
-plt.grid(True)
+ax.semilogx(t, np.exp(-t / 5.0))
+ax.grid()
 
 plt.show()
+
```
Comment (NelleV) [inline examples/pylab_examples/log_test.py]: I haven't looked at the travis log, but I wouldn't be surprised this blank line is a PEP8 violation.

Comment (choldgraf) [inline examples/pylab_examples/log_test.py]: I thought that it was the oppose - that blank lines at the end of a file are required?

Comment (choldgraf) [inline examples/pylab_examples/log_test.py]: since there needs to be a newline at the end?

Comment (choldgraf) [inline examples/pylab_examples/log_test.py]: looking at travis I think you're right.

Comment (dmgt) [inline examples/pylab_examples/log_test.py]: Thanks, I fixed it

**Comment by tacaswell** [comment]
@dmgt Thanks and congratulation on (what I think is) your first Matplotlib PR :tada:

**Comment by choldgraf** [comment]
wooo!!! way to go!! 🎉 🎉 🎉 🎉

**Pull request merged**